src/core/utils.js

```diff
@@ -130,7 +130,7 @@ function handleQuartetValues(value) {
 export {
   arrayToObject,
   calculateNewBackgroundPosition,
-  calculateNewTranslate,
+  flipTransformSign,
```

Comment:
I can't remember for certain, but I think that this would be a breaking change for folks because I'm pretty sure that this is part of the public API. cc @TxHawks who made these changes.

We may just want to keep the name as it was or do both:

```js
export {
  // ...
  flipTransformSign as calculateNewTranslate,
  flipTransformSign,
  // ...
}
```

Reply:
I'll add that alias back in!